Index: src/main/java/com/mycompany/student/email/Email.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/mycompany/student/email/Email.java b/src/main/java/com/mycompany/student/email/Email.java
deleted file mode 100644
--- a/src/main/java/com/mycompany/student/email/Email.java	
+++ /dev/null	
@@ -1,24 +0,0 @@
-//package com.mycompany.student.email;
-//
-//import org.springframework.beans.factory.annotation.Autowired;
-//import org.springframework.mail.SimpleMailMessage;
-////import org.springframework.mail.javamail.JavaMailSender;
-//import org.springframework.mail.MailSender;
-//import org.springframework.web.bind.annotation.RequestMapping;
-//import org.springframework.web.bind.annotation.RestController;
-//
-//@RestController
-//
-//public class Email {
-//    @Autowired
-//    private MailSender mailSender;
-//    @RequestMapping("/sendMail")
-//    public String sendConfirmationEmail() {
-//        SimpleMailMessage msg = new SimpleMailMessage();
-//        msg.setTo("swathi@email.com");
-//        msg.setSubject("message testing");
-//        msg.setText("Hello World from Spring Boot Email");
-//        mailSender.send(msg);
-//        return "OK";
-//    }
-//}
\ No newline at end of file
